src/components/form/GitSettingsForm.vue

```diff
@@ -1,112 +0,0 @@
-<template>
```

Comment:
I think you have to remove the test file too

Reply:
There was no test file related to `GitSettingsForm.vue`